pkg/cli/flags.go

```diff
@@ -360,6 +403,14 @@ func init() {
 		// We share the default with the ClientInsecure flag.
 		BoolFlag(f, &startCtx.serverInsecure, cliflags.ServerInsecure, startCtx.serverInsecure)
 
+		// Enable/disable various external storage endpoints.
+		// TODO(yevgeniy): currently we enable all schemas by default, but we should probably
```

Comment:
I'm not sure about this point about defaults -- I think we want s3/gcs/etc to "just work" out of the box, and I think the default is probably to have most of them on. Indeed, I'm sort of inclined to invert the struct fields to be `DisableFoo` form so the zero-value is everything on, since flipping one off, like HTTP, is expected to be the exception, not the rule?

I donno, maybe @mwang1026 has thoughts